app/src/store/preferencesStore.tsx

```diff
@@ -28,6 +28,17 @@ export interface PreferencesState extends PreferencesProps {
    * @returns
    */
   setTraceStreamingEnabled: (traceStreamingEnabled: boolean) => void;
+  /**
+   * Whether or not to show the span aside that contains details about timing, status, etc.
+   * @default true
+   */
+  showSpanAside: boolean;
+  /**
+   * Setter for enabling/disabling the span aside
+   * @param showSpanAside
+   * @returns
```

Comment:
i see i did the same above with empty @returns i guess that works since it returns void, i guess it just doesn't show up if there is no value after it